packages/ui/src/components/va-color-indicator/VaColorIndicator.vue

```diff
@@ -1,58 +1,53 @@
 <template>
-   <div
+  <div
     class="color-indicator"
-    @click="ctx.valueComputed = !ctx.valueComputed"
+    @click="valueComputed = !valueComputed"
     :class="computedClass"
     :style="computedStyle"
   >
     <div
       class="color-indicator__core"
-      :style="{'background-color': colorComputed, 'border-radius': square ? 0 : '50%'}"
+      :style="{ backgroundColor: colorComputed, borderRadius: square ? 0 : '50%' }"
```

Comment:
maybe we can do that?
`:style="{ backgroundColor: colorComputed, ...computedStyle }"`

Reply:
done